### ISSUE TITLE
Assert failed: fd >= 0

### ISSUE THREAD

**Issue by l29ah** [vgough/encfs#339]
Just happened to me during a tar backup, no idea how to reproduce.
```
Jun 28 02:15:01 [encfs] 2017-06-28 02:15:01,776 ERROR [default] Assert failed: fd >= 0
Jun 28 02:15:01 [encfs] 2017-06-28 02:15:01,778 ERROR [default] withFileNode: error caught in read: fd >= 0
Jun 28 02:15:01 [kernel] [166288.311324] encfs[8459]: segfault at 0 ip 000000000044f7a0 sp 0000039d937fd960 error 4 in encfs[400000+af000]
```

**Comment by rfjakob** [comment]
encfs version?

**Comment by l29ah** [comment]
On Mon, Jul 03, 2017 at 05:30:31PM +0000, rfjakob wrote:
> encfs version?

1.9.1

-- 
()  ascii ribbon campaign - against html mail
/\  http://arc.pasp.de/   - against proprietary attachments

**Comment by erikman** [comment]
Possible a  a duplicate of #214 where the `FileNode` gets corrupted?

**Comment by rfjakob** [comment]
Yes, possibly

**Comment by rfjakob** [comment]
@l29ah You are not running Arch Linux, are you?

**Comment by l29ah** [comment]
On Wed, Jul 05, 2017 at 09:36:40PM +0000, rfjakob wrote:
> @l29ah You are not running Arch Linux, are you?

No.

-- 
()  ascii ribbon campaign - against html mail
/\  http://arc.pasp.de/   - against proprietary attachments

**Comment by rfjakob** [comment]
@l29ah Thanks for the report. I believe this is fixed in v1.9.2, released right now.
